api/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.7'
```

Comment:
I think that, in the end, we will have a single `docker-compose` file that will have all services required to run the CHT, that will be built at runtime with the correct tags and upload to our staging server. 

This docker-compose file is for "dev" purposes?